mobile_verifier/src/coverage.rs

```diff
@@ -580,6 +691,7 @@ fn insert_indoor_coverage(
             seniority_timestamp: hex_coverage.coverage_claim_time,
             signal_level: hex_coverage.signal_level,
             hotspot: hotspot.clone(),
+            urbanized: hex_coverage.urbanized,
```

Comment:
You need to modify the query in `covered_hex_stream` to return the urbanized field

Reply:
:+1: